code/game/turfs/simulated/cleaning.dm

```diff
@@ -34,10 +34,13 @@
 
 //expects an atom containing the reagents used to clean the turf
 /turf/proc/clean(atom/source, mob/user)
+	var/amount = 0
+	amount = 0  // reset amount to 0 since the line above is done only once 
```

Comment:
wtf

Reply:
wtf

Reply:
I don't know man. The amount was kept between calls with only the first line.

I will try it again. I know it looks horrible.

Reply:
> I don't know man. The amount was kept between calls with only the first line.
> 
> I will try it again. I know it looks horrible.

try changing the variable name, amt for example.